### PR TITLE
C360556 Populating preview of matched records in case no matches

### PR DESCRIPTION
**Pull request by MaksimTerekhov**
https://foliotest.testrail.io/index.php?/cases/view/360556

### CODE REVIEW

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js [hunk 1]
```diff
@@ -49,5 +55,15 @@ describe('bulk-edit', () => {
       BulkEditActions.verifyCheckedDropdownMenuItem();
       BulkEditActions.verifyUncheckedDropdownMenuItem();
     });
+
+    it('C360556 Populating preview of matched records in case no matches (firebird)', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
+      BulkEditSearchPane.selectRecordIdentifier('User Barcodes');
+
+      BulkEditSearchPane.uploadFile(invalidUserBarcodesFileName);
+      BulkEditSearchPane.waitFileUploading();
+
```
Comment (penfriend) [inline cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:64]: I don't see here a check that there is no records fould from the step1: No records are matched and the label reads: 0 records match

Comment (MaksimTerekhov) [reply cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:66]: fixed

**Review by TetianaParanich — APPROVED** [review]

**Review by penfriend — APPROVED** [review]

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js [hunk 2]
```diff
@@ -36,6 +38,10 @@ describe('bulk-edit', () => {
       FileManager.deleteFile(`cypress/fixtures/${userBarcodesFileName}`);
```
Comment (penfriend) [inline cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:38]: Do you remove this file (cypress/fixtures/${invalidUserBarcodesFileName})  after the test execution ?

**Review by penfriend — CHANGES_REQUESTED** [review]

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js (hunk 2 above)
Comment (MaksimTerekhov) [reply cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js:38]: done

**Review by penfriend — APPROVED** [review]

**Pull request merged**